tower.yml

```diff
@@ -21,6 +21,16 @@ reports:
     display: "All samples Salmon merged transcript raw counts"
   "**/salmon/salmon.merged.transcript_tpm.tsv":
     display: "All samples Salmon merged transcript TPM counts"
+  "**/kallisto/**/deseq2.plots.pdf":
```

Comment:
Let's run a test of this in the platform to ensure these files are rendered before merging the PR.

Reply:
confirmed